README.rst

```diff
@@ -12,6 +12,24 @@ aioelasticsearch
 .. image:: https://codecov.io/gh/aio-libs/aioelasticsearch/branch/master/graph/badge.svg
   :target: https://codecov.io/gh/aio-libs/aioelasticsearch
 
+Getting Started
+---------------
+
+Aioelasticsearch - is an asynchronous elasticsearch python client wrapper.
+
+Prerequisites
+-------------
+
+Running Elasticsearch server is required.
+
+Run it with docker (Linux/MAC):
```

Comment:
MAC is Mandatory Access Control or Media Access Control. Linux is a kernel, not an OS.
```suggestion
Run it with docker (GNU/Linux or macOS):
```